Dockerfile

```diff
@@ -1,13 +1,13 @@
 # r7insight_docker
 
-FROM node:0.12-onbuild
+FROM node:12.13.0-buster
 LABEL maintainer="Rapid 7 - InsightOps Team <InsightOpsTeam@rapid7.com>"
 
 ENV NODE_ENV production
 
 WORKDIR /usr/src/app
-COPY package.json index.js VERSION ./
-RUN npm install --production
```

Comment:
Why remove `--production`?

Reply:
Specifying `NODE_ENV=production` is the same as `--production`